controller/translationcontroller.php

```diff
@@ -95,18 +95,18 @@ public function getLanguageStrings() {
 			'revision.restored' => $this->trans->t('Revision restored'),
 
 			// js/app/controllers/settings.js
-			'bookmarklet.text' => $this->trans->t('Save in passman'),
+			'bookmarklet.text' => $this->trans->t('Save in Passman'),
```

Comment:
Bookmarklet is a word with full meaning, what you purpose it's like calling a browser "C application that allows you to open web pages"